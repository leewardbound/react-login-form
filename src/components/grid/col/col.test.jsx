import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Col from '.';

const renderer = new ShallowRenderer();

describe('Col', function () {
  it('should render correctly without props', function () {
    const component = renderer.render(
      <Col>
        <h1>Lorem ipsum</h1>
      </Col>
    );

    expect(component).toMatchSnapshot();
  });

  it('should render correctly with props', function () {
    const component = renderer.render(
      <Col
        sm={12}
        md={6}
        mdOffset={6}
      >
        <h1>Lorem ipsum</h1>
      </Col>
    );

    expect(component).toMatchSnapshot();
  });
});
