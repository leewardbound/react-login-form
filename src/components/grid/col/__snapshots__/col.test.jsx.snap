// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col should render correctly with props 1`] = `
<div
  className="hDcSZq"
>
  <h1>
    Lorem ipsum
  </h1>
</div>
`;

exports[`Col should render correctly without props 1`] = `
<div
  className="hKVbzg"
>
  <h1>
    Lorem ipsum
  </h1>
</div>
`;
